object/user_util.go

```diff
@@ -80,6 +82,30 @@ func GetUserByFields(organization string, field string) (*User, error) {
 	return nil, nil
 }
 
+func IsPasswordSame(user *User, password string) (bool, error) {
+	organization, err := GetOrganizationByUser(user)
+	if err != nil {
+		return false, err
+	}
+
+	credManager := cred.GetCredManager(organization.PasswordType)
+
+	return credManager.IsPasswordCorrect(password, user.Password, user.PasswordSalt), nil
+}
+
+func CheckPasswordSame(user *User, password string, lang string) string {
```

Comment:
I think we should return error with "check:password must not be same as previous" from object layer and then wrap it with c.T in controller layer to divide controllers and "business" logic